### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by ruijietay**
1. Update content to provide an overview of ClickConnect.

1. Update the link of the GitHub Actions build status badge (Build Status) so that it reflects the build status of the ClickConnect repo.

1. Acknowledge the original source of the code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+# ClickConnect
+ClickConnect is an easy-to-use address book that is geared towards freelance photographers.
+Features such as the integration of a task list for each contact allows photographers
+to effectively track tasks with clients.
+
+The main way to interact with ClickConnect is via the keyboard. This allows photographers to quickly manage
+their clients, while still having a GUI that provides a visually pleasing experience to the user.
+
+* For the detailed documentation of this project, see the **[ClickConnect Product Website](https://ay2324s2-cs2103t-t17-3.github.io/tp/)**.
+* This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).
```
Comment (RyanNgWH) [inline README.md:14]: Should we add a link to the original project repository as well? We could link it to either the website or the github.

### Website
```suggestion
* This project is based on the [AddressBook-Level3](https://se-education.org/addressbook-level3) project created by the [SE-EDU initiative](https://se-education.org).
```

### Github Source
```suggestion
* This project is based on the [AddressBook-Level3](https://github.com/se-edu/addressbook-level3) project created by the [SE-EDU initiative](https://se-education.org).
```

**Review by RyanNgWH — COMMENTED** [review]
Looks good! Added a suggestion to link back to AB3 in the acknowledgement message

File: README.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/AY2324S2-CS2103T-T17-3/tp/actions)
 
 ![Ui](docs/images/Ui.png)
```
Comment (RyanNgWH) [inline README.md:3]: Do you think it is good to change this to our current mockup of the application?

**Review by RyanNgWH — COMMENTED** [review]
Added a suggestion to replace current UI diagram with our mockup

File: README.md (hunk 2 above)
Comment (ruijietay) [reply README.md:3]: Yes, that would be ideal. Thank you for the suggestion!

**Review by dillontkh — APPROVED** [review]
LGTM!

**Pull request merged**